sysutils/restic/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/restic/restic 0.9.4 v
+categories          sysutils
+
+license             BSD
+maintainers         nomaintainer
+
+description         Fast, secure, efficient backup program.
+
+long_description    Restic is a program that does backups right. Its design goals are: Easy, Fast, Verifiable, Secure, Efficient and Free.
+
+distname            restic-${version}
+homepage            https://restic.net/
+
+checksums           rmd160  05dde2739f3c2ed9109f1ece2e96008cec36ac2a \
+                    sha256  9198c43abd08b3a09ea59226282447316e13da579713dda2d81a28c37902d2c8 \
+                    size    26211387
+
+build.cmd           go run build.go
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
```

Comment:
It's recommended to also copy the LICENSE files (and other *.md, *.rst files) to destroot.

https://github.com/macports/macports-ports/blob/59c4aca854216604ddee9d60c5112cf4b4df7d3f/net/annie/Portfile#L21-L23

Reply:
No problem, these documents are usually not necessary. I'm just trying to make the port better ;)

Compiling the sphinx docs would be great, but that can be done later. Let's get this PR merged now. Thanks for your contribution!